src/Microsoft.ServiceFabric.Actors.KVSToRCMigration/TargetMigrationOrchestrator.cs

```diff
@@ -588,21 +594,33 @@ private async Task<IMigrationPhaseWorkload> NextWorkloadRunnerAsync(PhaseResult
         {
             var endSN = await this.GetEndSequenceNumberAsync(cancellationToken);
             var delta = endSN - currentResult.EndSeqNum;
+            var isDowntimeInvoked = false;
```

Comment:
NIT: Please use isDowntimeInvoked or IsDowntimeAllowed for local as well as meta data store., to avoid confusion later.

Reply:
Renamed all occurrences to isDowntimeInvoked